src/coreclr/vm/interpexec.cpp

```diff
@@ -159,14 +161,29 @@ static OBJECTREF CreateMultiDimArray(MethodTable* arrayClass, int8_t* stack, int
 #define LOCAL_VAR_ADDR(offset,type) ((type*)(stack + (offset)))
 #define LOCAL_VAR(offset,type) (*LOCAL_VAR_ADDR(offset, type))
 #define NULL_CHECK(o) do { if ((o) == NULL) { COMPlusThrow(kNullReferenceException); } } while (0)
+#define WITH_INLINED_CALL_FRAME(block) \
```

Comment:
I would prefer not creating a macro for this. Code in macros is a pain to debug, as you cannot step through the macro lines in the debugger. And since it is used at once place only and I don't expect more usages, it doesn't seem worth abstracting.